js/utils/constants.js

```diff
@@ -798,6 +798,47 @@
        */
       INITIAL_PLAY_STARTING: 'initialPlayStarting',
 
+      /**
+       * Notifies the player that the user has requested to play the previous video.
+       * Depending on the plugin being used, this could either be the previous video in
+       * a playlist, or the previously played Discovery video recommendation.
```

Comment:
Just out of curiosity, what happens when both playlists and discovery are used? Is this even a valid config?

Reply:
I need to confirm with Steve, but from my perspective this would not be a valid configuration. I will also confirm whether we need to do something about it. I think my current implementation wouldn't be able to handle this gracefully.